src/validators/IpValidator.php

```
@@ -212,28 +220,43 @@ public function init()
+        $this->hasSubnet ??= Yii::t(
+            'yii',
+            '{attribute} must not be a subnet.',
+        );
+        $this->notInRange ??= Yii::t(
+            'yii',
+            '{attribute} is not in the allowed range.',
+        );
+
+        if ((Yii::$app->useJquery ?? false) && !$this->clientScript instanceof ClientValidatorScriptInterface) {
+            $this->clientScript ??= ['class' => IpValidatorJqueryClientScript::class];
+            $this->clientScript = Yii::createObject($this->clientScript);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Yii' in method 'init'. (undefined)

(StaticAccess)